computation-local/src/main/java/com/powsybl/computation/local/LocalComputationManager.java

```diff
@@ -143,20 +144,42 @@ public OutputStream newCommonFile(String fileName) throws IOException {
     }
 
     private ExecutionReport execute(Path workingDir, List<CommandExecution> commandExecutionList, Map<String, String> variables, ExecutionMonitor monitor)
-            throws IOException, InterruptedException {
+            throws InterruptedException {
+        // TODO concurrent
         List<ExecutionError> errors = new ArrayList<>();
+        ExecutorService executionSubmitter = Executors.newCachedThreadPool();
 
         for (CommandExecution commandExecution : commandExecutionList) {
             Command command = commandExecution.getCommand();
+            CountDownLatch latch = new CountDownLatch(commandExecution.getExecutionCount());
+            IntStream.range(0, commandExecution.getExecutionCount()).forEach(idx ->
+                    executionSubmitter.execute(() -> {
+                        try {
+                            enter();
+                            logExecutingCommand(workingDir, command, idx);
+                            preProcess(workingDir, command, idx);
+                            int exitValue = process(workingDir, commandExecution, idx, variables);
+                            postProcess(workingDir, commandExecution, idx, exitValue, errors, monitor);
+                        } catch (Exception e) {
+                            LOGGER.warn(e.getMessage());
```

Comment:
Add the cause `e` to the log message to help debugging. Is WARN is the correct log level here?